string action_type
string object_id
geometry_msgs/Pose target_pose
string[] preconditions
string[] effects
